sys/ios-cydia.sh

```diff
@@ -18,9 +18,13 @@ export BUILD=1
 
 if [ ! -d sys/ios-include/mach/vm_behavior.h  ]; then
 (
-	cd sys && \
-	wget http://lolcathost.org/b/ios-include.tar.gz && \
-	tar xzvf ios-include.tar.gz
+	if [ ! -f sys/ios-include.tar.gz ]; then
+		set -e
+		cd sys && \
+		wget http://lolcathost.org/b/ios-include.tar.gz && \
+		set +e
+	fi
+	cd sys && tar xzvf ios-include.tar.gz
```

Comment:
what about untaring it after wgeting it? this may file if running it twice

Reply:
because the way i've done it, the file is not always downloaded, but it is directly embedded within r2 git repo, so the wget is not always done.